packages/rs-drive/src/drive/document/convert.rs

```diff
@@ -0,0 +1,30 @@
+use dpp::{document::document_stub::DocumentStub, prelude::Document};
+
+use crate::error::{drive::DriveError, Error};
+
+/// Helper function to convert DPP documents to Drive documents
+pub fn convert_dpp_documents_to_drive_documents<'a, I>(
+    dpp_documents: I,
+) -> Result<Vec<DocumentStub>, Error>
+where
+    I: Iterator<Item = &'a Document>,
+{
+    dpp_documents
+        .map(|document| {
+            DocumentStub::from_cbor(
+                &document.to_buffer().map_err(|_| {
+                    Error::Drive(DriveError::CorruptedCodeExecution(
+                        "Can't convert dpp document to CBOR",
+                    ))
+                })?,
+                None,
+                None,
+            )
```

Comment:
There is no need to do this conversion, Drive should only be using DocumentStubs (for now).